tests/kuttl/common/errors_cleanup_placement.yaml

```diff
@@ -70,20 +69,6 @@ spec:
     service: placement
   type: ClusterIP
 ---
-apiVersion: route.openshift.io/v1
-kind: Route
-metadata:
-  name: placement-public
-  labels:
-    public: "true"
-    service: placement
```

Comment:
the negative assertion requires this to be retained, and an error would be if the Route resource has been discovered. By dropping it, we just omit testing for No route existance

Reply:
As there is no Route created at all during the testing it is fairly moot to assert during cleanup that does not exist.

Reply:
oh, makes sense, I misunderstood